specs/glossary.md

```diff
@@ -125,14 +125,41 @@ verified against the Merkle root.
 
 ## Chain Re-Organization
 
+Note: As of the current state of this PR, it is possible for "reorg" and "replacement" to be used as interchangable terms.
+
 [reorg]: glossary.md#chain-re-organization
 
-A re-organization, or re-org for short, is whenever the head of a blockchain (its last block) changes (as dictated by
-the [fork choice rule][fork-choice-rule]) to a block that is not a child of the previous head.
```

Comment:
I found this line confusing previously because it seemed to indicate the parent<>child relationship of the blockchain gets broken when a reorg occurs, which is not the case. Rather, it's saying that while you might expect some *next* block to occur, the new block is actually replacing some block-height you've already seen, and so it is *not a child of the previous head*.

Reply:
I'd probably stick with the original text here.  I think the new version is a bit too ambiguous about whether adding a new block to the chain is a re-org (its not but it is an example of a node changing the head block). Although it should be "not a descendant of the previous head" - it's not a reorg to import 2 blocks at once.

Or to look at it another way, a re-org is when the canonical block at particular block number is replaced by a different block.

Reply:
> Or to look at it another way, a re-org is when the canonical block at particular block number is replaced by a different block.

Ah good catch, I indeed left open that natural chain-extension is implicated in my definition. I'll take how you've phrased it, it's simple and captures the definition.